deps/src/main/java/com/microsoft/azure/sdk/iot/deps/auth/IotHubCertificateManager.java

```diff
@@ -1,192 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-package com.microsoft.azure.sdk.iot.deps.auth;
```

Comment:
The point of this class was to hold the IoT Hub public certificate in source code. Since we no longer want to do that, this class can be deleted